help/zulip-cloud-billing.md

```diff
@@ -95,15 +114,20 @@ sponosorship](#apply-for-sponsorship). Contact
 
 ### Apply for sponsorship
 
-Submit a sponsorship request to apply for a free or discounted Zulip Cloud
-Standard plan. If you do not have a Zulip organization yet, contact
+Submit a sponsorship request to apply for a free or discounted **Zulip Cloud
+Standard** plan. If you do not have a Zulip organization yet, contact
 [sales@zulip.com](mailto:sales@zulip.com) to inquire about sponsorship
 eligibility.
 
 {start_tabs}
 
 {tab|request-sponsorship}
 
+1. Update your [organization type](/help/organization-type) to one of the
```

Comment:
Hm, I'm not sure this is helpful. At most, we should ask them to make sure their org type is accurate.

Have you been finding that it's common for the org type to need an update?

Reply:
Also, my vague recollection was we offer that as an option in the sponsorship application flow?

Reply:
Oh yeah, it can be selected in the sponsorship application. The field is pre-populated with the org type, at least on Zulip Cloud.

![Screenshot 2024-06-10 at 16 31 06@2x](https://github.com/zulip/zulip/assets/2090066/16a66a15-e1c9-4eb3-9444-6f38978a5ae4)